openedx/features/discounts/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+Discount API URLs
+"""
+from django.conf import settings
+from django.conf.urls import include, url
+
+from .views import CourseUserDiscount
+
+urlpatterns = [
+    url(r'^{}'.format(settings.COURSE_KEY_PATTERN), CourseUserDiscount.as_view(), name="discounts"),
```

Comment:
I'd expect this to have at least some prefix, rather than just the bare course id, both to indicate what the api is, and to allow url space for other api endpoints in this app.

Reply:
Fair.